Football/src/screens/SettingsScreen/types.ts

```diff
@@ -0,0 +1,8 @@
+export type RootState = {
+  settingsReducer: { isEnabledDarkTheme: boolean };
+};
+
+export type SettingsScreenViewPropsType = {
+  colorSchemeSwitchAction: () => any;
```

Comment:
why it returns any? please check that type.  Hover over the `colorSchemeSwitchAction` action